Trakt/Api/TraktApi.cs

```diff
@@ -639,7 +641,7 @@ public async Task<List<DataContracts.Users.Watched.TraktShowWatched>> SendGetWat
     /// <returns>Task{List{DataContracts.Users.Playback.TraktMoviePaused}}.</returns>
     public async Task<List<DataContracts.Users.Playback.TraktMoviePaused>> SendGetAllPausedMoviesRequest(TraktUser traktUser)
     {
-        return await GetFromTrakt<List<DataContracts.Users.Playback.TraktMoviePaused>>(TraktUris.PausedMovies, traktUser).ConfigureAwait(false);
+        return await GetFromTrakt<List<DataContracts.Users.Playback.TraktMoviePaused>>(TraktUris.PausedMovies + "?" + traktRandom.NextInt64(), traktUser).ConfigureAwait(false);
```

Comment:
```suggestion
        return await GetFromTrakt<List<DataContracts.Users.Playback.TraktMoviePaused>>(TraktUris.PausedMovies + "?" + traktRandom.NextInt64(), traktUser).ConfigureAwait(false);
```

The base url already has the beginnings of a querystring set

Reply:
I know but doing `&x=<random>` does not bypass the cache.

Reply:
Bypassing a cache seems super fishy

Reply:
I agree but there seems to be no other way since those endpoints dometimes didn't update for half an hour or more.
I doubt anyone should sync their stuff that often but it's nonetheless a problem...